lib/src/elastic_client_impl.dart

```diff
@@ -169,14 +169,14 @@ class Aggregation {
 }
 
 class Bucket {
-  dynamic key;
+  Object? key;
   int? docCount;
   Map<String, Aggregation>? aggregations;
 
-  Map toMap() {
+  Map<String, Object> toMap() {
```

Comment:
I don't see much value between `Map<String, Object>` and `Map<String, dynamic>`, and the later seems to be used in more places. Is there any reason to go with `Object`? With `dynamic`, we wouldn't need to always force the values to non-null.

Reply:
You are right, there isn't much of a difference. The culture here is to avoid using dynamic at all. Want me to change this?